### DISCUSSION TITLE
missing gpu nvtx tags in Nsight Systems UI

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1514] [Q&A]
Hi there,

I used `nsys` within the MONAI Core 1.2.0 container to run and profile the [fast_training_tutorial](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/fast_training_tutorial.ipynb).

For the **fast training part**, I found two issues when I tried to correlate the CPU-side `dataload` nvtx tag with the GPU side:
1. The  `dataload` nvtx tag is not showing in the GPU NVTX row.
2. The `RandCrop` nvtx tag is not showing in the GPU NVTX row. It should be correlated with the `dataload` nvtx tag, right?

Please see the screenshot below:
![image](https://github.com/Project-MONAI/tutorials/assets/90806807/db1ebea0-ae16-444b-9f19-b0980f038d8a)

Any idea?

**Comment by KumoLiu** [comment]
Hi @hw-ju, thanks for your interest here.

> The dataload nvtx tag is not showing in the GPU NVTX row.

- I didn't get your point for this one, I think the dataload is correctly shown in your screenshot under the processes folder.

> The RandCrop nvtx tag is not showing in the GPU NVTX row. It should be correlated with the dataload nvtx tag,

- In my experience, where Transform is located is sometimes really difficult to find, because it is not necessarily near dataload, you have to scroll down and open some collapsed folders to find it.

Hope it helps, thanks!
